### DISCUSSION TITLE
we were unable to verify your academic affiliation based on the information you submitted

### DISCUSSION THREAD

**Discussion by random-520** [community/community#48348] [GitHub Education]
I am a remote student. On February 21st, you sent an email telling me that I need to re-verify my education. I uploaded many different pictures but all failed. The picture contains my school name, my student ID, and my name. , photo, and date of enrollment. I don't know how to prove my academic affiliation? I need help with a human review, not a robot.

**Comment by jge162** [comment]
You should reach out to support if the bot is unable to verify you (as you mentioned). We can only discuss it here.

**Reply by random-520** [reply]
Thank you for your answer! I have contacted the support staff before this, but have not received a reply yet.

**Comment by samus-aran** [comment]
Hi @972793884 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

If you need help with applying please check out our [Apply to GitHub Global Campus for Students](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student) as well as our [Why wasn't my application to Global Campus for students approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved). We aim to assist our students by providing rejection reasons throughout the application process so that you can take steps to rectify and re-apply. 
You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by solokyo** [comment]
Have you passed your academic affiliation verification by requiring manual support?

**Comment by Abinashbunty** [comment]
@972793884 Using your **_academic email address_** will make you have a better edge for approval.

**Comment by shellyhdg** [comment]
<html>
<body>


Hey shellyhdg,Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Apr 24, 2023 for @shellyhdg.You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.
--



</body>
</html>Hey shellyhdg,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Apr 24, 2023 for [@shellyhdg](https://github.com/shellyhdg).

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

**Reply by samus-aran** [reply]
Hi @shellyhdg  👋

We’re happy you’re here but let’s not divert this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. 👍
